src/services/utilities.ts

```diff
@@ -857,6 +857,13 @@ namespace ts {
         return false;
     }
 
+    export function isInBreakOrContinue(sourceFile: SourceFile, position: number): boolean {
+        const contextToken = findPrecedingToken(position, sourceFile);
+        return contextToken && contextToken.parent &&

```

Comment:
If the kind matches, the parent will exist (since it's not a `SourceFile`), so no need to test for `&& contextToken.parent`.